src/intTest/resources/edifact/pathology.edifact.dat

```diff
@@ -41,12 +41,24 @@ RFF+STI:CH000037KA'
 DTM+SCO:201002231541:203'
 DTM+SRI:201002241541:203'
 GIS+N'
-INV+MQ+42R4.:911::Serum ferritin'
-RSL+NV+11++:::ng/mL'
+INV+MQ+:::B12/FOLATE'
+SEQ++1'
+FTX+SPC+++Results repeated:Low plasma Folate'
 STS++FI'
 RFF+ASL:1'
+GIS+N'
+INV+MQ+42T..:911::Serum vitamin B12'
+RSL+NV+1001++:::ng/L'
+STS++FI'
+RFF+ARL:1'
+S20+20'
+RND+U+170+1100'
+GIS+N'
+INV+MQ+42UE.:911::Plasma folate level'
+RSL+NV+0.8++:::ng/mL'
+RFF+ARL:1'
+STS++FI'
```

Comment:
Not sure where you've taken these examples from, but should `STS` be before `RFF`?

Reply:
I added the STS segments myself to test the mapping - just put them in the wrong order (our system doesn't care, but I agree - let's keep it correct!)